Bottom: f9e3e6c157c1ea2659d743602f87bb9412110c42
Top:    7667315eaae9ce07bd33ddfd9f5fddccdd43ade1
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2014-07-05 17:57:20 +0100

Refresh of attempt-at-supporting-arc-arc

---

diff --git a/src/polygon.c b/src/polygon.c
index 29b5a50..cf526de 100644
--- a/src/polygon.c
+++ b/src/polygon.c
@@ -652,7 +652,7 @@ ArcPolyNoIntersect (ArcType * a, Coord thick)
     return 0;
 
   if (rx == ry)
-    degree_circle (contour, a->X, a->Y, v, -a->Delta);
+    degree_circle (contour, a->X, a->Y, rx, v, -a->Delta);
   else
     for (i = 0; i < segs - 1; i++)
       {
@@ -674,7 +674,7 @@ ArcPolyNoIntersect (ArcType * a, Coord thick)
   v[0] = a->X - rx * cos (ang * M180);
   v[1] = a->Y + ry * sin (ang * M180);
   if (rx == ry)
-    degree_circle (contour, a->X, a->Y, v, a->Delta);
+    degree_circle (contour, a->X, a->Y, rx, v, a->Delta);
   else
     for (i = 0; i < segs; i++)
       {
